yuhow/Makefile

```diff
@@ -1,76 +0,0 @@
-SHELL := bash
```

Comment:
umm...maybe the reason was quite stupid. All the figures I had were pdf format. It was not possible to include pdf figure in Latex. So I just used PdfLatex instead of Latex. One thing after another, I also didn't know how to modify that Makefile. So I just directly abandoned that Makefile.